config/environments/production.rb

```diff
@@ -47,7 +47,7 @@
   # config.action_cable.allowed_request_origins = [ "http://example.com", /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
```

Comment:
Tại sao lại sửa thế này em?

Reply:
nó chuyển hướng trình duyệt http sang https ạ, và đặt cookies được đánh dấu là an toàn. Em sẽ comment lại ạ :v hiện tại chưa cần ạ.